docs/monthly-meeting/2022-07.md

```diff
@@ -0,0 +1,148 @@
+  - Dates: 16 Aug (pt. 1) & 18 Aug (pt. 2), both 4pm-6pm UTC
+
+## Agenda items
+
+### Past Action Items
+
+- [ ] CAM: Carol (and CAM?) will be updating the readme and guide at the sprints.
+  - Update: CAM didn't see any reviews or feedback on his PRs. (He did take some time off from Python work.)
+  - CAM: Petr ended up more or less doing this first, so I think we can check this off
+- [x] Adam: Creating a 'docs-approved' (wording to be agreed) label for CPython.
+  - https://github.com/python/core-workflow/issues/458
+  - Created as the `@python/proofreaders` team
+- [x] Petr: Start the Docs WG → Editorial Board rename
+  - https://github.com/python/docs-community/pull/55
+- [x] Petr: Solicit agenda items on Discourse for the next/future meetings
+  - https://discuss.python.org/t/16317
+- [x] Petr: Create a `proofreaders` team (Initially: CAM, Adam, Pradyun, Hugo), post about this team on Discourse for a wider audience
+  - https://discuss.python.org/t/16319
+- [x] Ezio: Add a “what's new” section to the devguide (key/important changes)
+  - https://github.com/python/devguide/issues/885
+  - Should we use blurb?
+  - Waiting for the devguide reorg
+- [ ] Ezio: Add a (How-To) Guides section to devguide
+  - Waiting for the reorg and still thinking about the best way to approach this :thinking_face:
+- [x] Mariatta: to work with Daniele to organize Diátaxis workshop, CAM & Ned to help with logistics.
+  - https://discuss.python.org/t/announcing-the-diataxis-documentation-workshop/17075
+
+
+### 'Internal' items
+
+*For and about the Community or Working Group*
+
+None
+
+### Diataxis Workshop
+* Many people signing up (currently oversubscribed), Mariatta will need to pick who can attend
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, &c.*
+
+* Writing Diataxis guidelines
+  * how to approach diataxification
+    * internal reorg, doc splitting, etc.
+  * making sections recognizable for readers and writers
+    * explicit boxes/directives
+    * naming conventions for sections
+    * naming conventions for anchors
+      * (i.e. ``.. _<module>-guide:``; ``.. _<module>-reference:``) 
+      * global summary page/table
+  * bunch-of-functions modules vs ~frameworks
+  * page structure
+    * initial paragraph with links vs ToC vs list of sections
+    * howtos section organization (titles/etc) / examples vs howtos
+
+* (Not) breaking URLs
+  * How much do we care?
+    - we should preserve reference links as much as possible. Tutorial/Guide/etc links we can try and preserve the path to the right location through preserving anchors (perhaps in the footer)
+  * Automation to prevent breakage?
+    * Apply data: Is it possible to look at web server logs to see which fragments are most often followed by incoming visitors?
+      * no, anchors aren't in server logs (unless we use JS tracking)
+    * [ ] Try building automation using the .inv file
```

Comment:
Recommend you take a look at my [sphobjinv](https://github.com/bskinn/sphobjinv) as possible machinery for this. Happy to field any questions.